spk/sonarr/Makefile

```diff
@@ -1,12 +1,12 @@
 SPK_NAME = nzbdrone
```

Comment:
I propose to use `Sonarr` as package name
Once we have to do the breaking change, why not now?

Reply:
When I looked into doing this previously there were some major issues that I didn't find an easy resolution for (at the time).

If I remember correctly a couple of the issues as a result of making this change were:

- By making this change synology package manager would now treat this as a new/separate package (would no longer use the standard package upgrade processes). This means that we would have to manually add a check for the old database and also build-in the database migration from the old app (via scripting). This was doable but added more potential problems than I wanted to deal with at the time.
- Synology package manager will not allow you to install a package with the same webports and a couple other items. This means that this package would require some scripting to work around that install issue. If I remember correctly you would need to have it start the install with different webports set in the new package, then remove the old app or change the webports via scripting as part of the process.
- In addition, and to potentially resolve the previous point, a bigger challenge (for me) was that I did not find a good way to add scripting to stop/uninstall the old app via scripting/commands within this new app. It seemed it would open up too many potential issues.

Perhaps there were more issues, I can't remember. However, I think this would be kind of nice to have if someone wants to make all of the above work. I don't know if another package has gone through this process before and might already have some of this scripting added to a package upgrade/rename. Another option is to make this a separate v3 package and force users to manually upgrade/migrate.

Reply:
Let's please not make this complicated just for a name that a regular users wouldn't see!

Reply:
> Let's please not make this complicated just for a name that a regular users wouldn't see!

That's what we decided last time and I agree

Reply:
I have to look at the documentation but in DSM7 there is a way to replace a package wich is exactly what we need to address these issues. We just need to wait until it's released

Reply:
Also agree that for this PR we should leave it as is. It's not worth work on on until DSM7 is released